pages/manager_page.py

```diff
@@ -0,0 +1,70 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+from pages.locators import AllLocators
+
+
+class ManagerPage:
+    def __init__(self, browser):
+        self.browser = browser
+        self.url = "https://www.globalsqa.com/angularJs-protractor/BankingProject/#/manager"
+
+    def open(self):
+        self.browser.get(self.url)
+        element_present = EC.visibility_of_element_located((By.XPATH, "//button[@ng-class='btnClass1']"))
+        WebDriverWait(self.browser, 5).until(element_present)
+
+    def open_add_customers_form(self):
+        self.browser.find_element(*AllLocators.ADD_CUSTOMER_OPEN_FORM_BUTTON).click()
+        element_present = EC.visibility_of_element_located((By.XPATH, "//input[@ng-model='fName']"))
+        WebDriverWait(self.browser, 5).until(element_present)
```

Comment:
Здесь и далее. Может стоит сократить код? Предложил бы даже создать базовый метод для взаимодействия с элементами страниц на основе `WebDriverWait`. То есть мы не будем ожидать в конкретном методе появления каких-либо других элементов (в этом просто нет смысла), а будем взаимодействовать с конкретным, то есть WebDriverWait будет ожидать появления `ADD_CUSTOMER_OPEN_FORM_BUTTON` и далее нажимать на данный элемент - кнопку.
```suggestion
        WebDriverWait(self.browser, 5).until(EC.visibility_of_element_located(*AllLocators.ADD_CUSTOMER_OPEN_FORM_BUTTON)).click()
```